crypto/src/main/java/io/warp10/crypto/OSSKeyStore.java

```diff
@@ -91,17 +87,14 @@ public byte[] decodeKey(String encoded) {
     if (null == encoded) {
       return null;
     }
-    if (encoded.startsWith("hex:")) {
-      return Hex.decode(encoded.substring(4));
-    } else if (encoded.startsWith("base64:")) {
-      return Base64.decode(encoded.substring(7));
-    } else if (encoded.startsWith("wrapped:")) {
+
+    if (encoded.startsWith("wrapped:")) {
       if (null == this.masterKey) {
         throw new RuntimeException("Master Key not retrieved from OSS, aborting.");
       }
-      return CryptoHelper.unwrapBlob(this.masterKey, decodeKey(encoded.substring(8)));
+      return CryptoHelper.unwrapBlob(this.masterKey, CryptoUtils.decodeKey(this, encoded.substring(8)));
     } else {
-      return null;
+      return CryptoUtils.decodeSimpleKey(encoded);
```

Comment:
This function does not handle argon2. Is this intentional?

Reply:
It was with the original signatures. Updated to support argon2.